pkgs/build-support/setup-hooks/patch-shebangs.sh

```diff
@@ -1,9 +1,11 @@
-# This setup hook causes the fixup phase to rewrite all script
-# interpreter file names (`#!  /path') to paths found in $PATH.  E.g.,
-# /bin/sh will be rewritten to /nix/store/<hash>-some-bash/bin/sh.
-# /usr/bin/env gets special treatment so that ".../bin/env python" is
-# rewritten to /nix/store/<hash>/bin/python.  Interpreters that are
-# already in the store are left untouched.
+# This setup hook patches installed scripts to add Nix store paths to their
```

Comment:
🤦🏻‍♂️Yes, forgot that. But how would you reference the manual properly
a) from that character blob of a script
b) to a possibly ever-changing document structure?

I'd not be happy with "see the manual for how to use this".

Also while it would introduce double book keeping, documentation to that script should be right there, and also visible in the manual for anyone not wanting to dig into the code. And I would not see the point of adding more mechanism to auto-document shell scripts right now.

I'll remove the change and open an issue to discuss how to approach this.